axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveLineServiceImpl.java

```diff
@@ -1141,7 +1145,8 @@ private void populateMoveLineMap(
   }
 
   @Override
-  public void autoTaxLineGenerate(Move move) throws AxelorException {
+  @Transactional
```

Comment:
Need to manage the rollback

https://redmine.axelor.com/projects/axerp/wiki/Caveats#Transactions